bitcoin/src/network/constants.rs

```diff
@@ -104,6 +104,17 @@ impl Network {
     pub fn magic(self) -> Magic {
         Magic::from(self)
     }
+
+    /// Converts a `Network` to its equivalent bitcoind `-chain` option argument name.
+    /// Bitcoind documentation refers to the following allowed options: `-chain=[main, test, signet, regtest]`
```

Comment:
I went looking for how you found this and came up with these docs, use them if you like them :)
```suggestion
    /// Converts a `Network` to its equivalent `bitcoind -chain` argument name.
    ///
    /// ```bash,no_run
    /// $ bitcoin-23.0/bin/bitcoind --help | grep -C 3 '\-chain=<chain>'
    /// Chain selection options:
    ///
    /// -chain=<chain>
    /// Use the chain <chain> (default: main). Allowed values: main, test, signet, regtest
    /// ```
```

Reply:
Appreciate it! much cleaner than what I had going on haha